uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-jgit/src/main/java/org/uberfire/java/nio/fs/jgit/daemon/ssh/GitSSHService.java

```diff
@@ -91,14 +135,20 @@ public void setup(final File certDir,
         checkNotNull("repositoryResolver",
                      repositoryResolver);
 
-        sshd.getProperties().put(SshServer.IDLE_TIMEOUT, sshIdleTimeout);
+        buildSSHServer(gitSshCiphers,
+                       gitSshMacs);
+
+        sshd.getProperties().put(SshServer.IDLE_TIMEOUT,
+                                 sshIdleTimeout);
 
         if (inetSocketAddress != null) {
             sshd.setHost(inetSocketAddress.getHostName());
             sshd.setPort(validateOrGetNew(inetSocketAddress.getPort()));
 
             if (inetSocketAddress.getPort() != sshd.getPort()) {
-                LOG.error("SSH for Git original port {} not available, new free port {} assigned.", inetSocketAddress.getPort(), sshd.getPort());
+                LOG.error("SSH for Git original port {} not available, new free port {} assigned.",
```

Comment:
avoid change format where you don't need

Reply:
ok